infra/terraform/test-org/org/locals.tf

```diff
@@ -26,98 +26,393 @@ locals {
     "ci-shared"   = module.folders-root.ids["ci-shared"]
   }
 
-  ci_folders = module.folders-ci.ids
-  ci_repos_folders = merge(
-    { for m in local.tgm_org_modules : m => { folder_name = "ci-${m}", folder_id = replace(module.folders-ci.ids["ci-${m}"], "folders/", ""), gh_org = "terraform-google-modules" } },
-    { for m in local.gcp_org_modules : m => { folder_name = "ci-${m}", folder_id = replace(module.folders-ci.ids["ci-${m}"], "folders/", ""), gh_org = "GoogleCloudPlatform" } }
-  )
-  tgm_org_modules = [
-    "kms",
-    "network",
-    "folders",
-    "slo",
-    "sap",
-    "iam",
-    "event-function",
-    "kubernetes-engine",
-    "vpn",
-    "project-factory",
-    "pubsub",
-    "migrate",
-    "bootstrap",
-    "datalab",
-    "jenkins",
-    "container-vm",
-    "lb",
-    "vm",
-    "memorystore",
-    "service-accounts",
-    "cloud-storage",
-    "sql-db",
-    "vpc-service-controls",
-    "cloud-datastore",
-    "dataflow",
-    "cloud-nat",
-    "startup-scripts",
-    "scheduled-function",
-    "address",
-    "bigquery",
-    "bastion-host",
-    "org-policy",
-    "log-export",
-    "cloud-dns",
-    "gsuite-export",
-    "terraform-validator",
-    "lb-http",
-    "gcloud",
-    "lb-internal",
-    "utils",
-    "composer",
-    "github-actions-runners",
-    "healthcare",
-    "gke-gitlab",
-    "example-foundation", # Not module
-    "anthos-platform",    # Not module
-    "cloud-operations",
-    "cloud-foundation-training", # Not module
-    "cloud-router",
-    "group",
-    "data-fusion",
-    "module-template",
-    "vault",
-    "docs-samples",
-  ]
+  ci_folders       = module.folders-ci.ids
+  ci_repos_folders = { for repo in local.repos : try(repo.short_name, trimprefix(repo.name, "terraform-google-")) => { folder_name = "ci-${try(repo.short_name, trimprefix(repo.name, "terraform-google-"))}", folder_id = replace(module.folders-ci.ids["ci-${try(repo.short_name, trimprefix(repo.name, "terraform-google-"))}"], "folders/", ""), gh_org = repo.org } }
```

Comment:
nit: it would be good to split this into multiple lines for readability

Reply:
SG